core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -252,7 +244,8 @@ private object RTS {
           case a: IO.Redeem[_, _, _, _] =>
             errorHandler = a.err.asInstanceOf[Any => IO[Any, Any]]
           case f0: Finalizer =>
-            val f: IO[Nothing, Option[List[Throwable]]] = fork(f0.finalizer, _ => IO.unit).observe.map(collectDefect)
+            val f: IO[Nothing, Option[List[Throwable]]] =
+              fork(f0.finalizer, _ => IO.unit).observe.map((a: ExitResult[Nothing, Unit]) => a.exceptions)
```

Comment:
would `.map(_.exceptions)` work too?

Reply:
For some reason compiling throws a `missing parameter type for expanded function` without it.